ttt/lib/ttt/board.ex

```diff
@@ -1,16 +1,19 @@
 defmodule TTT.Board do
+  @x_mark "X"
```

Comment:
Not sold on how 'critical' these constants are. 
You use them in exactly one function.
Plus, id aim for `:x` and `:o`, as strings are purely for the UI.